pkg/controller/ratelimiter/ratelimiter.go

```diff
@@ -65,10 +65,12 @@ func NewRateLimiter() ratelimiter.RateLimiter {
 // traffic to 5 qps.  That will hopefully leave enough capacity for
 // more latency sensitive reconciliations, at the expense of a longer
 // delay in re-reconciliation.
-func RequeueRateLimiter() ratelimiter.RateLimiter {
-	return workqueue.NewMaxOfRateLimiter(
-		// 5 qps, 50 bucket size.  This is the overall factor, and must be slower than the NewRateLimiter limit, to leave "room" for new items.
-		&workqueue.BucketRateLimiter{Limiter: rate.NewLimiter(rate.Limit(5), 50)},
+func RequeueRateLimiter() workqueue.TypedRateLimiter[reconcile.Request] {
+	// 5 qps, 50 bucket size.  This is the overall factor, and must be slower than the NewRateLimiter limit, to leave "room" for new items.
+	return workqueue.NewTypedMaxOfRateLimiter(
+		workqueue.NewTypedItemExponentialFailureRateLimiter[reconcile.Request](2*time.Second, 600*time.Second),
```

Comment:
Did you mean to change this one?

Reply:
Good question. Fairly sure I had something in mind but sadly I did not document and now I'm less convinced. Going to revert this line. If later I decide we need it after all, better it be in a separate PR.

Reply:
make ready-pr did not add them back so they should be gone.